_observing-your-data/alerting/api.md

```diff
@@ -1342,6 +1342,19 @@ Introduced 1.0
 
 Returns an array of all alerts.
 
+**Query Parameters**
+
+Parameter | Data type | Description
+:--- | :--- | :---
+sortOrder     | string | Determines the order of the results. can be "asc" or "desc". defaults to "asc"
```

Comment:
Global comment:
In parameter and request/response field tables, let's use the back ticks to set off parameter and field names in code style: `sortOrder`
For an example, see the parameters in [Query string queries](https://opensearch.org/docs/latest/query-dsl/full-text/query-string/).

Suggestion:
If you add one space before and one space after the parameter name, they'll render just find on the website; the column widens to the longest entry and proper spaces are added after the shorter entries.

| `sortOrder` | String | Determines the order of the results. Options are `asc` or `desc`. Defaults to `asc`. |